model/test/email/attachment_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http_parser/http_parser.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:model/email/attachment.dart';
+
+void main() {
+  final accountId = AccountId(Id('1'));
+  const baseDownloadUrl = 'http://localhost/download/{accountId}/{blobId}/{name}?name={name}&type={type}';
+
+  group('attachment test:', () {
+    group('getDownloadUrl:', () {
```

Comment:
please write more about the case 
- name include ? (Fx: `Subject has question mark ?`)
- Unicode name ? (Fx: `Đây là tiêu đề thư`)
- ...

Reply:
Done